src/main/groovy/com/codebrig/arthur/observe/structure/filter/conditional/ElseConditionalFilter.groovy

```diff
@@ -17,20 +18,32 @@ import com.google.common.collect.Sets
  */
 class ElseConditionalFilter extends StructureFilter<ElseConditionalFilter, Void> {
 
-    private static final Set<String> conditionalTypes = new HashSet<>()
-    static {
-        conditionalTypes.add("If") //python
-        conditionalTypes.add("IfStmt") //go
-        conditionalTypes.add("IfStatement") //java, javascript
-    }
+    private final MultiFilter elseConditionalFilter
+
     private final Set<Integer> elseNodeIdentities = Sets.newConcurrentHashSet()
 
+    ElseConditionalFilter() {
+        super()
+        this.elseConditionalFilter = createElseConditionalFilter()
+    }
+
+    private static createElseConditionalFilter() {
+        MultiFilter elseStatementFilter = MultiFilter.matchAll(
```

Comment:
Please combine all of these filters. Each filter should use at max one MultiFilter. To do *logical or* type logic use:
```groovy
MultiFilter.matchAll(
        new RoleFilter("IF"), new RoleFilter("STATEMENT", "EXPRESSION")
)
```

This means it must have (`IF`) and (`STATEMENT` or `EXPRESSION`).